package.json

```diff
@@ -119,6 +119,23 @@
           "default": "",
           "description": "The path to your elm executable. Should be empty by default, in that case it will assume the name and try to first get it from a local npm installation or a global one. If you set it manually it will not try to load from the npm folder."
         },
+        "elmLS.elmReviewPath": {
+          "scope": "window",
+          "type": "string",
+          "default": "",
+          "description": "The path to your elm-review executable. Should be empty by default, in that case it will assume the name and try to first get it from a local npm installation or a global one. If you set it manually it will not try to load from the npm folder."
+        },
+        "elmLS.elmReviewDiagnostics": {
+          "scope": "window",
+          "type": "string",
+          "enum": [
+            "off",
+            "warning",
+            "error"
+          ],
+          "default": "warning",
+          "description": "Set severity or disable linting diagnostics for elm-review."
```

Comment:
I choose a different text, as I think it might be better. Maybe we want this in the other places to. But it doesn't make sense to list the possible values here, while it does in the other places.